packages/circuit-ui/components/Anchor/Anchor.spec.tsx

```diff
@@ -84,15 +85,14 @@ describe('Anchor', () => {
     it('should call the onClick handler when rendered as a link', async () => {
       const props = {
         ...baseProps,
-        'href': 'https://sumup.com',
-        'onClick': jest.fn((event: ClickEvent) => {
+        href: 'https://sumup.com',
+        onClick: jest.fn((event: ClickEvent) => {
           event.preventDefault();
         }),
-        'data-testid': 'anchor',
       };
-      const { getByTestId } = renderAnchor(render, props);
+      renderAnchor(render, props);
 
-      await userEvent.click(getByTestId('anchor'));
+      await userEvent.click(screen.getByRole('link'));
```

Comment:
Shall we migrate the other tests in this file to use `screen` as well?

Reply:
Here you go 😛 cb81075